pkg/kubelet/cm/containermap/container_map.go

```diff
@@ -69,3 +71,19 @@ func (cm ContainerMap) GetContainerRef(containerID string) (string, string, erro
 	}
 	return cm[containerID].podUID, cm[containerID].containerName, nil
 }
+
+// ListContainerRef lists all container ref pair from container map,
+func (cm ContainerMap) ListContainerRef() map[string]sets.String {
```

Comment:
I'm not a fan of this function that returns an unintuitive data structure like this. If we want to add something that walks all of the entries in the container map, I'd be fine with that though. Something like:
```
func (cm ContainerMap) Visit(f func(podUID, containerName, containerID string)) {
	for k, v := range cm {
		f(v.podUID, v.containerName, k)
	}
}
```
Which (following your code below) could then be called as:
```
cm.Visit(func(podUID, containerName, containerID string) {
	if _, ok := activeContainers[podUID][containerName]; !ok {
		klog.ErrorS(nil, "RemoveStaleState: removing container", "podUID", podUID, "containerName", containerName)
		err := m.policyRemoveContainerByRef(podUID, containerName)
		if err != nil {
			klog.ErrorS(err, "RemoveStaleState: failed to remove container", "podUID", podUID, "containerName", containerName)
		}
	}
})
```

Reply:
appreciate. change to visit, PTAL @klueska